runhouse/resources/hardware/cluster.py

```diff
@@ -1946,3 +1954,75 @@ def _folder_mv(
 
     def _folder_exists(self, path: Union[str, Path]):
         return self.client.folder_exists(path=path)
+
+    ###############################
+    # Cluster list
+    ###############################
+    @classmethod
+    def list(
+        cls,
+        show_all: Optional[bool] = False,
+        since: Optional[str] = None,
+        status: Optional[ResourceServerStatus] = None,
+    ):
```

Comment:
I think this returns:
```suggestion
    ) -> Dict[str, List[Dict]] :
```